server/player/player.go

```diff
@@ -2208,7 +2246,13 @@ func (p *Player) Tick(w *world.World, current int64) {
 		}
 	}
 
-	p.checkBlockCollisions(w)
+	if _, ok := p.Armour().Chestplate().Item().(item.Elytra); ok && p.Gliding() {
+		if t := p.glideTicks.Inc(); t%20 == 0 {
+			p.armour.SetChestplate(p.damageItem(p.Armour().Chestplate(), 1))
```

Comment:
We should probably force the player to stop gliding if the elytra breaks here.

Reply:
Done.